frontend/src/views/WheresMyGene/components/HeatMap/components/YAxisChart/index.tsx

```diff
@@ -159,10 +202,44 @@ export default memo(function YAxisChart({
         )}
       </TissueWrapper>
       <Container
-        data-test-id="cell-type-labels"
+        data-test-id={`cell-type-labels-${tissueKey}`}
         height={heatmapHeight}
         ref={yAxisRef}
       />
+      {yAxisInfoCoords &&
+        yAxisInfoCoords.map((coord, i) => {
+          const content = cellTypeMetadata[i];
+          return (
+            <div
+              id={`${content}`}
+              key={`${content}`}
+              style={{
+                position: "absolute",
+                left: coord[0],
+                top: coord[1],
+                cursor: "pointer",
+              }}
+              onClick={() => {
+                if (isMarkerGenes) {
+                  const { id } = deserializeCellTypeMetadata(content);
+
+                  generateMarkerGenes({
+                    cellTypeID: id,
+                    organismID: selectedOrganismId,
+                    tissueID,
+                  });
+                }
+              }}
+            >
+              <Image
+                src={InfoSVG.src}
+                width="9"
+                height="9"
+                alt="display marker genes"
```

Comment:
nice alt 🔥 ! Should we go the extra mile and add cell type name if it's easy 😆 ?